ngraph/test/visitors/op/floor.cpp

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "gtest/gtest.h"
+
+#include "ngraph/ngraph.hpp"
+#include "ngraph/op/util/attr_types.hpp"
+#include "ngraph/opsets/opset1.hpp"
+#include "util/visitor.hpp"
+
+using namespace std;
+using namespace ngraph;
+using ngraph::test::NodeBuilder;
+
+TEST(attributes, floor_op)
+{
+    NodeBuilder::get_ops().register_factory<opset1::Floor>();
+    auto A = make_shared<op::Parameter>(element::f32, Shape{5, 2});
+
+    auto floor = make_shared<opset1::Floor>(A);
+    NodeBuilder builder(floor);
+    auto g_floor = as_type_ptr<opset1::Floor>(builder.create());
+
+    EXPECT_EQ(g_floor->get_autob(), floor->get_autob());
```

Comment:
Floor doesn't read autobroadcast and in spec such operation doesn't have any attributes.

Reply:
Oh good point, why this test passed?

Reply:
Test passed because [AutoBroadcastSpec ](https://github.com/openvinotoolkit/openvino/blob/master/ngraph/core/include/ngraph/node.hpp#L201) is part of the Node class which was inherited by Floor. I've replaced this test with check for 0 arguments [here](https://github.com/openvinotoolkit/openvino/pull/5618/commits/ab165d7457df04009da780a30cddc061bb2e2977)